csp90.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Cyberslot's official site",
+  "subdomain": "csp90",
+  "owner": {
+    "repo": "ytcyberslotp90/Cyber-Slot-official-website",
```

need github username
```suggestion
    "username": "ytcyberslotp90",
```